packages/model-viewer/src/three-components/Model.ts

```diff
@@ -231,11 +227,12 @@ export default class Model extends Object3D {
   clear() {
     this.url = null;
     this.userData = {url: null};
+    const gltf = this[$currentGLTF];
     // Remove all current children
-    if (this[$currentScene] != null) {
-      moveChildren(this.modelContainer, this[$currentScene]!);
-      this[$currentScene]![$releaseFromCache]();
-      this[$currentScene] = null;
+    if (gltf != null) {
+      moveChildren(this.modelContainer, gltf.scene);
```

Comment:
Moving children around like this is probably not necessary; Three.js allows a `Scene` to be a child of another `Object3D`, so we could probably just take advantage of that and insert the `gltf.scene` directly into the graph. Punting for the future, though.